### PR TITLE
Pluralize: Allows to override config

### PR DESCRIPTION
**Pull request by ngonzalez**
Requires https://github.com/svenfuchs/i18n/pull/373 in order to fix CI

### CODE REVIEW

File: lib/i18n/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module I18n
-  VERSION = "0.8.4"
+  VERSION = "0.8.4-be3fe"
```
Comment (radar) [inline lib/i18n/version.rb]: Please undo this change.

**Comment by radar** [comment]
Code looks good to me. I am not clear on how this is used. Do you have an example where this would be useful?

**Comment by ngonzalez** [comment]
@radar Hello, thanks for reviewing this PR. I updated the branch with latest changes from master, and added a comment that explains the use of `plural_keys`. We already use this fork at my company website, in production.

**Comment by ngonzalez** [comment]
^ build is failing but it's affecting `master` branch too

**Comment by ngonzalez** [comment]
@radar Hello 👋 Any chances you take a look at the changes again?

**Comment by radar** [comment]
I sure can take a look again :)

I just tried this in an app with both `i18n` and `rails-i18n` installed. I was unable to reproduce any issue, and I believe that was because `rails-i18n` [defines pluralization rules for the `zh` locale](https://github.com/svenfuchs/rails-i18n/blob/719b6dba1c519edfb5dea0d1b80db77750d6f483/rails/pluralization/zh.rb).

So when I do this in a `rails console` session for that app, I have no problems:

```
irb(main):001:0> I18n.t(:test, locale: :zh, count: 1)
=> "test: other 1"
irb(main):002:0> I18n.t(:test, locale: :zh, count: 2)
=> "test: other 2"
```

I therefore strongly encourage you to add the `rails-i18n` gem to your app, or at least add these pluralization rules from that gem to your app.

I believe this patch is now no longer necessary and so I will be closing this issue.

**Comment by ngonzalez** [comment]
Thanks for reviewing, but this was for non-rails applications.

**Comment by ngonzalez** [comment]
I was able to set pluralization rules for `:zh` locale this way:

```
I18n::Backend::Simple.include(I18n::Backend::Pluralization)
I18n.load_path += ["locales/pluralization.rb"]
```

`locales/pluralization.rb`
```
{:zh => 
  { :i18n => 
    { :plural => 
      { :keys => [:other],
        :rule => Proc.new { :other }
      } 
    } 
  } 
}
```

Again, thanks for your time for the review and the useful comments :+1:

**Comment by radar** [comment]
Yes that is exactly right :) glad you were able to figure it out!
